### DISCUSSION TITLE
Link para o GitHub com as resoluções dos exercícios propostos

### DISCUSSION THREAD

**Discussion by MiguelLopes2002** [raguirreleal/Git-Curso-FURG#8] [General]
#
### Link do Miguel:
https://github.com/MiguelLopes2002/Exercicio-Final.git
#

**Comment by Costa95** [comment]
Link da Camila:

https://github.com/Costa95/Exercicios

**Comment by LarissaCastagna** [comment]
Link Larissa:
https://github.com/LarissaCastagna/Exercicios-finais.git

**Comment by Verislaine** [comment]
Link Verislaine
https://github.com/Verislaine/exercicio-final

**Comment by brunafpinho** [comment]
link bruna: https://github.com/brunafpinho/exercicio-final-
